api/src/model/play_asset_pack_states.gd

```diff
@@ -0,0 +1,49 @@
+# ##############################################################################
+#
+#	Copyright 2020 Google LLC
+#
+#	Licensed under the Apache License, Version 2.0 (the "License");
+#	you may not use this file except in compliance with the License.
+#	You may obtain a copy of the License at
+#
+#		https://www.apache.org/licenses/LICENSE-2.0
+#
+#	Unless required by applicable law or agreed to in writing, software
+#	distributed under the License is distributed on an "AS IS" BASIS,
+#	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#	See the License for the specific language governing permissions and
+#	limitations under the License.
+#
+# ##############################################################################
+#
+# Wraps Play Core's AssetPackStates which represents the states for a 
+# collections of pack.
+#
+# ##############################################################################
+class_name PlayAssetPackStates
+extends Object
+
+var _total_bytes : int
+var _pack_states : Dictionary
+
+func _init(init_dictionary : Dictionary):
+	_total_bytes = init_dictionary["totalBytes"]
```

Comment:
Thanks for breaking out the dictionary values into `var`s. Should we do the same for the other type (e.g. PlayAssetPackState, PlayAssetPackLocation etc.)?

Reply:
I spent some time thinking about this; Although the data structure for other types are not as complexity, I think it is still better to declare separate var for them (makes code clearer and corrupted Dictionary error can be found during init)

Reply:
Applied this modification to all other classes.